AssetStoreBatchMode/AssetStoreBatchMode.cs

```diff
@@ -129,7 +132,7 @@ public static void UploadAssetStorePackage(params string[] args)
     /// <param name="loginTimeout">The maximum amount of time to wait (in seconds) when logging in. Defaults to 90 seconds. Must be within 2 and 36000 seconds. Login is attempted twice.</param>
     /// <param name="metadataTimeout">The maximum amount of time to wait (in seconds) when getting metadata. Defaults to 600 seconds. Must be within 2 and 36000 seconds.</param>
     /// <param name="uploadTimeout">The maximum amount of time to wait (in seconds) when uploading. Defaults to 36000 seconds. Must be within 2 and 36000 seconds.</param>
-    public static void UploadAssetStorePackage(string username, string password, string packageName, string rootPath = null, string[] mainAssets = null, int loginTimeout = 90, int metadataTimeout = 600, int uploadTimeout = 36000, bool skipProjectSettings = false)
+    public static void UploadAssetStorePackage(string username, string password, string packageName, string rootPath = null, string[] mainAssets = null, GuidListPostprocessCallback guidListPostprocessCallback = null, int loginTimeout = 90, int metadataTimeout = 600, int uploadTimeout = 36000, bool skipProjectSettings = false)
```

Comment:
Can we add the callback as an event on the class itself?

This method is typically called in batch mode (from the command line) so the callback argument wouldn't be possible there.

Reply:
Just saw this, my apologies you're correct here. Had been a while since I looked at our build scripts, we're using environment variables.

It could possibly be nice to _represent_ the callback somehow via command line (passing another method name), but I don't think that's worth the extra complexity at all.